src/mmw/.jshintrc

```diff
@@ -6,6 +6,7 @@
     "browser": true,
     "curly": true,
     "eqeqeq": true,
+    "esnext": true,
```

Comment:
Ugh, I was a little wary about it but I hoped it wouldn't be a problem. There have been multiple time with this app where just a basic `let` would have made something easier and/or more readable.

Is there a list of browsers this app is required to support? I checked the compatibility of ES6/`let` and it looked reasonable at first glance.